### ISSUE TITLE
Support simultaneous keys of 3 or more

### ISSUE THREAD

**Issue by lizyn** [yqrashawn/GokuRakuJoudo#52]
When setting `<from>` to simultaneous keys as combination of more than two keys (in simple syntax, e.g. `[:a :b :c]`), goku 0.2.9 yield error messages:
```
ERROR: invalid <from> in main section's XXX(rule description)
Failed!
```

I know that most Apple keyboards have only "2-key rollovers", but a number of three-key-combo or 4-key-combo, etc. can still be recognized by certain keyboards. And `karabiner.json` supports that syntax, so I think it's reasonable to adjust this behavior.

**Comment by narze** [comment]
It works if you use `:rules` format I've tried 4 sim keys

```
{:des "SuperDuper <- S+D" :rules [:!qmk [{:sim [:s :d]
                                     :simo {:interrupt true
                                            :dorder :insensitive
                                            :uorder :insensitive
                                            :afterup {:set ["superduper-mode" 0]}}} {:set ["superduper-mode" 1]}]]}
        {:des "A+S+D -> SD+Option" :rules [:!qmk [{:sim [:a :s :d]
                                     :simo {:interrupt true
                                            :dorder :insensitive
                                            :uorder :insensitive
                                            :afterup {:set ["superduper-mode" 0]}}} [{:set ["superduper-mode" 1]} :left_option]]]}
        {:des "A+S+D+F-> SD+Delete word" :rules [:!qmk [{:sim [:a :s :d :f]
                                     :simo {:interrupt true
                                            :dorder :insensitive
                                            :uorder :insensitive
                                            :afterup {:set ["superduper-mode" 0]}}} [{:set ["superduper-mode" 1]} :!Odelete_or_backspace]]]}
```

**Comment by yqrashawn** [comment]
fixed in v0.3.0
